lib/pharos/scripts/configure-etcd.sh

```diff
@@ -83,9 +94,7 @@ EOF
 fi
 
 echo "Waiting etcd to launch on port 2380..."
-
-while ! nc -z ${PEER_IP} 2380; do
+while ! etcd_healthy; do
```

Comment:
I think we should put some timeouts (5min or something long enough) around these sleep loops